### PR TITLE
Migrate test_replica3_to_arbiter

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrate the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py)

Updates: #292

Fixes: #775

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
```sh

TestName : test_replica3_to_arbiter
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:03:03       |      NA     |
|   dist-rep  |     PASS    |        00:01:19       |      NA     |
+-------------+-------------+-----------------------+-------------+
```

File: tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                                                      counter, 2, 2, 2, 10,
+                                                      mount['client'])
+            self.io_process.append(proc)
+            counter = counter + 10
+
+        # Convert relicated to arbiter volume
+        self._convert_replicated_to_arbiter_volume()
+
+        # Wait for IO to complete.
+        ret = redant.wait_for_io_to_complete(self.io_process,
```
Comment (nik-redhat) [inline tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py:137]: I know, both the functions are same but better to use the `validate_io_procs` to keep consistency.

File: tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        # Verify volume's all process are online
+        if not (self.redant.
+                verify_all_process_of_volume_are_online(self.vol_name,
+                                                        self.server_list[0])):
+            raise Exception(f"All volume processes for {self.vol_name}"
+                            " are not online")
+
+    def run_test(self, redant):
+        """
+        Description: Replica 3 to arbiter conversion with ongoing IO's
```
Comment (nik-redhat) [inline tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py:100]: nit: can be removed

File: tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py]: Ack

File: tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py (hunk 2 above)
Comment (aujjwal-redhat) [reply tests/functional/arbiter/brick_cases/test_replica3_to_arbiter.py]: Ack

**Comment by aujjwal-redhat** [comment]
```sh

✔ Environment setup successful.
✔ Running test case : test_replica3_to_arbiter-rep
✔ test_replica3_to_arbiter-rep Succeeded
✔ Running test case : test_replica3_to_arbiter-dist-rep
✔ test_replica3_to_arbiter-dist-rep Succeeded
Results for brick_cases
Test nature : disruptive
TestName : test_replica3_to_arbiter
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:02:53       |      NA     |
|   dist-rep  |     PASS    |        00:01:11       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**